rotkehlchen/chain/ethereum/modules/zksync/decoder.py

```diff
@@ -67,6 +68,18 @@ def _decode_deposit(self, context: DecoderContext, user_address: ChecksumEvmAddr
 
         return DEFAULT_DECODING_OUTPUT
 
+    def _decode_withdrawal(self, context: DecoderContext) -> DecodingOutput:  # noqa: E501
+        """Decode zksync lite withdrawal event."""
```

Comment:
Can you also mention in the docstring that the log event doesn't contain information about the withdrawn amount or token since there are several withdrawals bundled?